crates/nu-command/tests/commands/cp.rs

```diff
@@ -7,16 +7,42 @@ use nu_test_support::nu;
 use nu_test_support::playground::Playground;
 use std::path::Path;
 
+fn get_file_hash<T: std::fmt::Display>(file: T) -> String {
+    nu!("open {} | to text | hash md5", file).out
+}
```

Comment:
FYI, this is a bit expensive. Not gonna say we shouldn't do it this way, but `nu!` launches a new instance of Nushell and that's a lot slower than a function call. Probably OK to leave as is for now though.